### ISSUE TITLE
Keep only async flow (suspend fun) by removing sync code

### ISSUE THREAD

**Issue by osoykan** [Trendyol/kediatR#26]

**Comment by CanerPatir** [comment]
Hello Oğuzhan, 
Sync command bus behaviors had been located to support places where don't supply async manner like Spring Boot @KafkaListener. However, we can handle sync command bus calls over Kotlin's out of the box runSync dispatcher. So It would be great enhancement @osoykan

**Comment by bilal-kilic** [comment]
I aggre this is a great suggestion to declutter the project.
Maybe we can leave sync implementations on `CommandBusImpl` to provide an easy way to run sync commands.

```
override fun <TCommand : Command> executeCommandSync(command: TCommand) = processPipeline(registry.getPipelineBehaviors(), command) {
    runBlocking {
        registry.resolveCommandHandler(command.javaClass).handle(command)
    }
}
```

**Comment by osoykan** [comment]
Implemented in v2.0
